jcl/source/common/JclExprEval.pas

```diff
@@ -980,6 +980,7 @@ implementation
   Windows, // inline of AnsiSameText
   {$ENDIF ~HAS_UNITSCOPE}
   {$ENDIF SUPPORTS_INLINE}
+  Math,
```

Comment:
This one should be in the "HAS_UNITSCOPE" section, with its prefixed counterpart in IF part